tests/grid_test.cc

```diff
@@ -96,17 +97,10 @@ void testGrid( const GridView& gridView )
         std::cout << "number of elements is wrong: " << numElem << "\n";
 }
 
-int main()
+template <class Grid>
+void testGrid(Grid& grid, const std::string& name)
 {
-    Opm::Parser parser;
-    Opm::ParseMode parseMode;
-    const auto deck = parser.parseString(deckString , parseMode);
-
-    std::vector<double> porv;
-    typedef Dune::PolyhedralGrid< 3, 3 > Grid;
-    typedef Grid::LeafGridView GridView;
-    Grid grid(deck, porv);
-
+    typedef typename Grid::LeafGridView GridView;
 #if DUNE_VERSION_NEWER(DUNE_GRID,3,0)
```

Comment:
If the indexInInside() issue can easily be fixed (without breaking the concept of the class...) please do so. If this is not a quick fix we should discuss what do do.

I still wonder if the 3.0 should be a 2.4 in the #if above?